packages/www/components/Plans/index.tsx

```diff
@@ -594,33 +484,18 @@ const Plans = ({ dashboard = false, stripeProductId }: PlanProps) => {
                       justifyContent: "center",
                       alignItems: "center",
                     }}>
-                    <span>500,000 minutes</span>
-                    <Tooltip
-                      multiline
-                      content="Then $0.50 for every additional 1,000 minutes">
-                      <Flex
-                        css={{
-                          borderRadius: 1000,
-                          bc: "$sage12",
-                          color: "$loContrast",
-                          width: 18,
-                          height: 18,
-                          fontSize: "$1",
-                          alignItems: "center",
-                          justifyContent: "center",
-                          cursor: "default",
-                          ml: "$2",
-                        }}>
-                        $
-                      </Flex>
-                    </Tooltip>
+                    <span>$100 minimum spend * </span>
                   </Flex>
                 }
               />
               <Item
-                css={{ borderBottom: 0, fontSize: "11px" }}
+                css={{
+                  borderBottom: 0,
+                  fontSize: "11px",
+                  justifyContent: "center",
+                }}
                 displayCheck={false}
-                title={"* Pay as you go past allotted minutes"}
+                title={"* (we round up if below $100)"}
```

Comment:
I think this could be rephrased more clearly. Rounding sounds like you pay $100 if you spent $99.01.

Maybe?
```suggestion
                title={"* Monthly pay of at least $100 or more depending on your usage"}
```
(the parenthesis seem redundant too, this is already a `*` remark)

Reply:
It's exactly how it is! from now on, if you spend 1 or 99$ you still pay 100$ monthly at least for the `Growth` plan. The phrasing in the frontend was given to me by hunter from the livepeer.studio pricing page that is coming